packages/core/lib/event/PluginInterface.ts

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
Why is this file in this PR? Didn't we add this in a different PR?

Reply:
Yes, your right, remnant of the merge.
Removed it.